firefox/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OctoLinker",
+  "version": "4.1.1",
```

Comment:
I would say no, because there is no chrome version 4.1.1 available. In future they will be always in sync, I guess.

Regarding the `version` field in the `package.json`. I think we can get rid of it completely since this isn't a npm module.